client/src/pages/Login.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react'
+import { Link, useNavigate } from 'react-router-dom'
+import { gql, useMutation } from '@apollo/client'
+
+export const SIGN_IN_MUTATION = gql`
+  mutation signIn($phone: String!, $password: String!) {
+    signIn(input: { phone: $phone, password: $password }) {
+      token
+    }
+  }
+`
+
+const Login = () => {
+  const [phone, setPhone] = useState<string>('')
+  const [password, setPassword] = useState<string>('')
+  const navigate = useNavigate()
+
+  const [signInData, { loading }] = useMutation(SIGN_IN_MUTATION)
+
+  const handleSignIn = (e: React.FormEvent) => {
+    e.preventDefault()
+
+    signInData({
+      variables: { phone, password },
+    })
+      .then((response) => {
+        const data = response.data.signIn
+        if (data.token) {
+          localStorage.setItem('token', data.token)
+          navigate('/')
+        } else {
+          alert('You have entered a wrong phone number or password')
+        }
+      })
+      .catch((error) => {
+        console.error('Error signing in:', error)
```

Comment:
Remove console logs from your code.
Any error messages can be shown in a graceful way on the UI.